lib/spree/shipment_notice.rb

```diff
@@ -60,7 +60,6 @@ def ship_it!
       shipment.update_attribute(:tracking, tracking)
 
       unless shipment.shipped?
-        shipment.reload.ready! if shipment.pending?
```

Comment:
From tests after capture_payments shipment state is set to ready, so is not really doing something